firmware/library/L1_Drivers/pwm.hpp

```diff
@@ -13,6 +10,4 @@ class PWMInterface
         virtual float GetDutyCycle() = 0;
```

Comment:
Aren't you missing a `SetDutyCycle()` function?

Reply:
Also, it would be great if you have one that accepts a float.

Reply:
I believe I have that function on line 9.